docs/level_0/first_simple_example.md

```diff
@@ -0,0 +1,209 @@
+# First simple Example
+
+In this example we describe how to deploy an nginx with the help of Landscaper.
+
+We describe how to develop a component containing a Component Descriptor and a Blueprint with a DeployItem.
+The DeployItem specifies how to deploy the nginx as a helm chart.
+
+## Deploy Item
+
+We want to deploy an nginx via the helm chart [nginx ingress](https://github.com/kubernetes/ingress-nginx/tree/master/charts/ingress-nginx).
+Therefore, we specify a DeployItem in a yaml file with the following content.
+
+```yaml
+deployItems:
+- name: nginx
+  type: landscaper.gardener.cloud/helm                                                             # (1)
+  target:
+    name: {{ index .imports "target-cluster" "metadata" "name" }}                                  # (2)
+    namespace: {{ index .imports "target-cluster" "metadata" "namespace" }}                        # (3)
+  config:
+    apiVersion: helm.deployer.landscaper.gardener.cloud/v1alpha1
+    kind: ProviderConfiguration
+
+    chart:
+      ref: "eu.gcr.io/gardener-project/landscaper/tutorials/charts/ingress-nginx:v0.1.0"           # (4)
+
+    updateStrategy: patch
+
+    name: nginx                                                                                    # (5)
+    namespace: first-example                                                                       # (6)
+```
+
+Since we want to deploy a helm chart, the type of the DeployItem is set to `landscaper.gardener.cloud/helm` in line (1).
+
+The fields in lines (2) and (3) are not yet specified. They are bound to the import parameter `target-cluster` of the 
+Blueprint, that we will define below. The target section defines the name and namespace of a kubernetes custom resource
+containing the access data for the target cluster on which the nginx will be deployed. We leave it variable, in order 
+to make the Deploy Item reusable. So it can be used to deploy the nginx on different clusters. 
+
+Line (4) contains a reference to the helm chart for the nginx. The helm chart is stored in an OCI registry because 
+Landscaper is currently not able to fetch helm charts from helm chart repositories.
+
+Line (5) and (6) define the name of the nginx helm installation, and the namespace into which it will be deployed. In a 
+real world scenario it would be a good idea to provide the value of the namespace as import data to get a better 
+reusable component.
+
+You find the DeployItem in file [deploy-execution-nginx.yaml](./resources/blueprint/deploy-execution-nginx.yaml).
+
+## Blueprint
+
+Next, we specify the Blueprint. The Blueprint consists of a [directory](./resources/blueprint) containing a 
+[blueprint.yaml](./resources/blueprint/blueprint.yaml) with the following content, and the 
+[deploy-execution-nginx.yaml](./resources/blueprint/deploy-execution-nginx.yaml) of the DeployItem. 
```

Comment:
Clarify: deploy-execution-nginx.yaml from the previous step.

Start with a tree view of the folder. Then first mention the deploy-execution-nginx.yaml from the previous step and then describe blueprint.yaml.

Reply:
we added a tree view of the folder